tests/testthat/_snaps/response_process.md

```diff
@@ -1,86 +1,124 @@
+      * reason: SERVICE_DISABLED
+      * domain: googleapis.com
+      * metadata.consumer: projects/977449744253
+      * metadata.service: sheets.googleapis.com
 
 # Request with invalid argument (Sheets, bad range)
 
-    Client error: (400) INVALID_ARGUMENT
-    * Client specified an invalid argument. Check error message and error details for more information.
-    * Unable to parse range: NOPE!A5:F15
+    Code
+      response_process(resp)
+    Condition
+      Error in `expect_recorded_error()`:
+      ! Client error: (400) INVALID_ARGUMENT
+      * Client specified an invalid argument. Check error message and error details for more information.
+      * Unable to parse range: NOPE!A5:F15
 
 # Request with bad field mask (Sheets)
 
-    Client error: (400) INVALID_ARGUMENT
-    * Client specified an invalid argument. Check error message and error details for more information.
-    * Request contains an invalid argument.
-    
-    Error details:
-    Field violations
-    * Field: sheets.sheetProperties
-      Description: Error expanding 'fields' parameter. Cannot find matching fields for path 'sheets.sheetProperties'.
+    Code
+      response_process(resp)
+    Condition
+      Error in `expect_recorded_error()`:
+      ! Client error: (400) INVALID_ARGUMENT
+      * Client specified an invalid argument. Check error message and error details for more information.
+      * Request contains an invalid argument.
+      
+      Error details:
+      Field violations
+      * Field: sheets.sheetProperties
+        Description: Error expanding 'fields' parameter. Cannot find matching fields for path 'sheets.sheetProperties'.
 
 # Request for nonexistent resource (Sheets)
 
-    Client error: (404) NOT_FOUND
-    * A specified resource is not found, or the request is rejected by undisclosed reasons, such as whitelisting.
-    * Requested entity was not found.
+    Code
+      response_process(resp)
+    Condition
+      Error in `expect_recorded_error()`:
+      ! Client error: (404) NOT_FOUND
+      * A specified resource is not found, or the request is rejected by undisclosed reasons, such as whitelisting.
+      * Requested entity was not found.
 
 # Request with invalid value (tokeninfo, stale token)
 
-    Client error: (400) Bad Request
-    * Invalid Value
+    Code
+      response_process(resp)
+    Condition
+      Error in `expect_recorded_error()`:
+      ! Client error: (400) Bad Request
+      * Invalid Value
 
 # Request to bad URL (tokeninfo, HTML content)
 
-    Expected content type 'application/json', not 'text/html'.
-    * Not Found
+    Code
+      response_process(resp)
+    Condition
+      Error in `expect_recorded_error()`:
+      ! Client error: (404) Not Found
+      x Expected content type 'application/json', not 'text/html'.
+      i See 'VOLATILE_FILE_PATH' for the html error content.
+      i Or execute `browseURL("VOLATILE_FILE_PATH")` to view it in your browser.
```

Comment:
This is a look at the new handling for an error that comes as html. In certain contexts, such as RStudio, the "See 'VOLATILE_FILE_PATH' ..." is a clickable link to open the file. I can't make the `browseURL()` a clickable link because it calls into a base package. But the user can copy it, paste into the console, and execute without too much trouble.

https://cli.r-lib.org/reference/links.html#security-considerations